app/src/main/java/com/ryan/heroestopbuilds/MainActivity.java

```diff
@@ -212,18 +252,31 @@ public String onChildPress(String name) {
      * @return final string for DB
      */
     public String prettyPrinter(ArrayList<String> popularSkills) {
-        //add our final list to a new list to be passed to MainActivity
+        ArrayList<String> finalList = new ArrayList<>();
         String lgSpacing = String.format("%" + 3 + "s", "");
         String smSpacing = String.format("%" + 1 + "s", "");
-        ArrayList<String>finalList = new ArrayList<>();
-        for(int i = 0; i <= levelMod; ++i) {
-            if(i <= 2) {
-                finalList.add("Level " + (1 + 3 * i) + ": " + lgSpacing + popularSkills.get(i));
-            } else {
-                finalList.add("Level " + (1 + 3 * i) + ": " + smSpacing + popularSkills.get(i));
+        //Check first skills, this means we got a Chromie
```

Comment:
Lets fuck this shit up, FAM.

Seeing that popularSkills is an ArrayList we can lean on pattern matching the strings within. This'll blow your fuckin mind.

for(String s : popularSkills) {
if(s.matches("(?i)(CompoundingAether | DeepBreath | TimewalkersPursuit | PeerIntoTheFuture)")){
isChromie_? = 1
}
}

I would assume those skill names can be shortened if no other heroes have matching skill names or combos of words